types/messages_encoding.go

```diff
@@ -122,3 +122,153 @@ func (s *SSVMessage) HashTreeRootWith(hh ssz.HashWalker) (err error) {
 func (s *SSVMessage) GetTree() (*ssz.Node, error) {
 	return ssz.ProofTree(s)
 }
+
+// MarshalSSZ ssz marshals the SignedSSVMessage object
+func (s *SignedSSVMessage) MarshalSSZ() ([]byte, error) {
+	return ssz.MarshalSSZ(s)
+}
+
+// MarshalSSZTo ssz marshals the SignedSSVMessage object to a target array
+func (s *SignedSSVMessage) MarshalSSZTo(buf []byte) (dst []byte, err error) {
```

Comment:
ok...
So we need to decide how to do this...

In #345 we added the `SignedSSVMessage` but we merged it to the fork branch because of this very encoding change...
I made this decision because I decided that the ssz encoding is anyhow the correct (consistent) way to go and I didn't take into account that this PR would need this.

I think we should rework #345 to remove the ssz encoding and add flat encoding so no fork

Reply:
Removed `SignedSSVMessage` from this PR.